### DISCUSSION TITLE
Correct that deadlines BT-131 and BT-1311 apply to both Other single stage procedure and Other multiple stage procedure?

### DISCUSSION THREAD

**Discussion by Colan71** [OP-TED/eForms-SDK#781] [Q&A] [answered]
Dear All,
Looking at the rules for "BT-131(d)-Lot Deadline Receipt Tenders" and "BT-1311(d)-Lot Deadline Receipt Requests" in field.json 

Is it really correct that the procedure types Other single stage procedure and Other multiple stage procedure apply to both of these deadline fields? Shouldn't BT-131 only apply to Other single stage procedure and BT-1311 only apply to Other multiple stage procedure?

Parts from the fields.json:

**BT-131(d)-Lot**
   "noticeTypes" : [ "16", "17", "18", "20", "21", "22", "23", "24" ],
        "condition" : "{ND-LotTenderingProcess} ${BT-105-Procedure == 'open' or (BT-105-Procedure == 'oth-mult' and (BT-1311(d)-Lot is not present)) or (BT-105-Procedure == 'oth-single' and (BT-1311(d)-Lot is not present))}",
        "value" : true,
        
**BT-1311(d)-Lot**
    "noticeTypes" : [ "16", "17", "18", "20", "21", "22", "23", "24" ],
        "condition" : "{ND-LotTenderingProcess} ${(BT-105-Procedure == 'oth-mult' and (BT-131(d)-Lot is not present)) or (BT-105-Procedure == 'oth-single' and (BT-131(d)-Lot is not present))}",
        

Best regards,
Åsa

**Comment by YvesJo** [comment]
Hi Asa,
Indeed these rules aren't strict enough and will be updated.

**Reply by YvesJo** [reply, marked as answer]
More restrictive rules will be applied starting SDK 1.11
